docs/recipes/github-actions.md

```diff
@@ -4,8 +4,6 @@
 
 The [Authentication](../usage/ci-configuration.md#authentication) environment variables can be configured with [Secret variables](https://help.github.com/en/articles/virtual-environments-for-github-actions#creating-and-using-secrets-encrypted-variables).
 
-For this example, you'll need to setup a `NPM_TOKEN` to publish your package to NPM registry and a `GH_TOKEN` to generate a release at GitHub.
-
```

Comment:
I would leave the first part, the `NPM_TOKEN` still needs to be created (It can be generated at `https://www.npmjs.com/settings/[USERNAME]/tokens/create`)

Reply:
Good! I changed it. I also added links to documentation about how to generate these tokens.